Milestone 3/backend/index.ts

```diff
@@ -0,0 +1,29 @@
+import {Express} from "express"
+import recipeRouter from "./routes/recipe";
+import { Recipe } from "./models/recipeSchema"
+const mongoose = require("mongoose")
+const express = require("express") // 1. includes Express 
+const app: Express = express() // 2. initializes Express 
+
+console.log("Backend running")
+app.use(express.json());
+
+const url = "mongodb+srv://admin:Password123!@cluster0.8zcta2q.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
I am getting an authentication error when trying to connect to your cluster with this url. Can you verify that you used the correct password here and that it's working on your end?